client/client.go

```diff
@@ -14,20 +14,22 @@ import (
 	// Useful for debug messages, or string manipulation for datastore keys.
 	"strings"
 
+	// Useful for formatting strings
+	"fmt"
+
 	// Want to import errors.
 	"errors"
 
 	// Optional. You can remove the "_" there, but please do not touch
 	// anything else within the import bracket.
 	_ "strconv"
-	// if you are looking for fmt, we don't give you fmt, but you can use userlib.DebugMsg.
```

Comment:
Does anyone recall the reasoning for not giving students the `fmt` package in the past?